src/main.ts

```diff
@@ -2,8 +2,13 @@ import { TOKENS, get } from '@/services'
 import App from './app/App.vue'
 
 async function mountVueApplication() {
+  const env = get(TOKENS.env)
+  const kumaApi = get(TOKENS.api)
+  kumaApi.baseUrl = env('KUMA_API_URL')
```

Comment:
**Comment**: Setting the base URL outside of the bootstrap function allows for re-using it with a different base URL.